pages/feed/index.tsx

```diff
@@ -62,7 +62,10 @@ export default function Feeds(props: HighlightSSRProps) {
   const topRef = useRef<HTMLDivElement>(null);
   const singleHighlight = props.highlight;
   const ogImage = props?.highlight
-    ? `${process.env.NEXT_PUBLIC_OPENGRAPH_URL}/highlights/${props.highlight.id}`
+    ? `${new URL(
+        `/og-images/highlight/${props.highlight.id}`,
+        process.env.NEXT_PUBLIC_BASE_URL || "http://localhost:3000"
+      )}`
```

Comment:
Testing locally, I get a 404 on: http://localhost:3000/og-images/highlight/301

Is this the right endpoint to hit?

Reply:
You need to run `netlify dev` ([Netlify CLI](https://docs.netlify.com/cli/get-started/)) if testing OG images locally.

Reply:
🫠 gotcha! Thanks Nick!

Reply:
Here's a deploy preview URL: https://deploy-preview-3779--oss-insights.netlify.app/og-images/highlight/298